Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -328,6 +328,31 @@ def condition(self, fragment, stepList, key, hasHarvest):
     offset = 0.9,
 )
 
+# Track DNN workflows
+
+class UpgradeWorkflow_trackdnn(UpgradeWorkflow):
+    def setup_(self, step, stepName, stepDict, k, properties):
+        stepDict[stepName][k] = merge([{'--procModifiers': 'trackdnn'}, stepDict[step][k]])
+	custNew = "RecoTracker/FinalTrackSelectors/customiseForDNN.customiseForDNN"
```

Comment:
looks like whitespace is messed up here, could be the reason why the test failed